src/Components/TwinSearchDropdown/TwinSearchDropdown.styles.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    ITwinPropertySearchDropdownStyleProps,
+    ITwinPropertySearchDropdownStyles
+} from './TwinSearchDropdown.types';
+
+/**
+ * Twin search dropdown styles
+ */
+export const classPrefix = 'cb-twin-search-dropdown';
+const classNames = {
+    root: `${classPrefix}-root`,
+    dropdown: `${classPrefix}-dropdown`,
+    label: `${classPrefix}-label`,
+    requiredIcon: `${classPrefix}-requiredIcon`
+};
+
+export const getStyles = (
+    props: ITwinPropertySearchDropdownStyleProps
+): ITwinPropertySearchDropdownStyles => {
+    const { theme } = props;
+    return {
+        root: [classNames.root],
+        dropdown: [classNames.dropdown],
+        label: [classNames.label, { alignItems: 'center', display: 'flex' }],
+        requiredIcon: [
+            classNames.requiredIcon,
+            {
+                '::after': {
+                    color: 'var(--cb-color-text-error)',
```

Comment:
Will this require callouts/modals that have this dropdown to be wrapped in base component?

Reply:
hmm yes. I think so. But only if they show the label. Right now we don't use the label so it's not an issue in the Advanced Search.

Reply:
We don't currently have hte error color on the palette anywhere else. :/ It's probably the most common thing that is missing.